content/index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Hello World</title>
+  <script src="//code.jquery.com/jquery-1.11.3.min.js"></script>
```

Comment:
It's used here for vertical centering of the content:

```
$("document").ready(function(){
    $(".outerbox").css({"height": ($(window).height() - 20) + "px"});
});
```

It could probably be done differently, but this is the faster way I can come up with.